documentation/modules/deploying/con-deploy-product-downloads.adoc

```diff
@@ -5,12 +5,12 @@
 [id='downloads-{context}']
 = Downloading Strimzi release artifacts
 
+[role="_abstract"]
 ifdef::Downloading[]
-To install Strimzi, download the release artifacts from {ReleaseDownload}.
+To use deployment files to install Strimzi, download the release artifacts from {ReleaseDownload}.
 endif::Downloading[]
-
 ifndef::Downloading[]
-To install Strimzi, download and extract the release artifacts from the `amq-streams-__<version>__-ocp-install-examples.zip` file from the {ZipDownload}.
+To use deployment files to install Strimzi, download and extract the release artifacts from the `amq-streams-__<version>__-ocp-install-examples.zip` file from the {ZipDownload}.
```

Comment:
Can we use some placeholders here instead of referring to AMQ Streams?

Reply:
Yep. I changed to _download and extract the installation and example files from the {ZipDownload}_

Reply:
```
[role="_abstract"]
ifdef::Downloading[]
To use deployment files to install Strimzi, download the release artifacts from {ReleaseDownload}.

Download the `strimzi-_<version>_.zip` or `strimzi-_<version>_.tar.gz` file archive.
endif::Downloading[]
ifndef::Downloading[]
To use deployment files to install Strimzi, download and extract the *{ProductName} _<version>_ installation and example files* archive from the {ZipDownload}.
endif::Downloading[]
```